docs/src/main/asciidoc/security-keycloak-authorization.adoc

```diff
@@ -309,21 +323,6 @@ docker run --name keycloak \
 <1> For `keycloak.version`, ensure the version is `26.0.7` or later.
 <2> For Keycloak keystore, use the `keycloak-keystore.jks` file located at https://github.com/quarkusio/quarkus-quickstarts/blob/main/security-keycloak-authorization-quickstart/config/keycloak-keystore.jks[quarkus-quickstarts/security-keycloak-authorization-quickstart/config].
 
-
```

Comment:
Why was this block removed ?

Reply:
Thanks for asking. I removed this block because I realized it was a duplicate of lines 327–344, likely introduced during an earlier revision.